nwbwidgets/timeseries.py

```diff
@@ -966,4 +942,485 @@ def update(
         fig = fig if fig is not None else go.FigureWidget()
         fig.data = []
         fig.layout = {}
-        return self.plot_group(group_inds,data,time_ts_aligned,fig,order)
+        return self.plot_group(group_inds, data, time_ts_aligned, fig, order)
+
+
+def get_time_series_data(
```

Comment:
Could you use this function and adjust it as necessary?

https://github.com/NeurodataWithoutBorders/nwbwidgets/blob/dba14008af7f7aea6f3ea83f010a0a89cca1bfa4/nwbwidgets/utils/timeseries.py#L86

Reply:
On it.

Reply:
Done.